infrastructure/modules/serverless/data.tf

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-data "aws_kms_key" "this" {
-  key_id = "alias/${var.namespace}-kms-key"
-}
+# data "aws_kms_key" "this" {
```

Comment:
Do we need this or not? If not remove it

Reply:
Removed